packages/core/src/components/axes/toolbar.ts

```diff
@@ -101,12 +101,27 @@ export class Toolbar extends Component {
 				.each(function (d: any, index: number) {
 					select(this)
 						.select('svg')
+						.style('pointer-events', 'none')
 						.style('will-change', 'transform')
 						.style('width', d.iconSVG.width !== undefined ? d.iconSVG.width : '20px')
 						.style('height', d.iconSVG.height !== undefined ? d.iconSVG.height : '20px')
 
 					select(this)
 						.select('button')
+						.on('mouseover focus', function (event: MouseEvent) {
+							const hoveredElement = select(this)
+							hoveredElement.classed('hovered', true)
+							self.services.events.dispatchEvent(Events.Toolbar.SHOW_TOOLTIP, {
+								event,
+								hoveredElement,
+								content: d.title,
+								noWrap: true,
+								placements: ['top', 'bottom']
```

Comment:
Hi @theiliad,

Based on the reviews, now it would put candidate placements for `.findBestPlacement` to determine.